### ISSUE TITLE
Search autocomplete different than search result page

### ISSUE THREAD

**Issue by kevinvuillemin** [Smile-SA/elasticsuite#1525]
Hello all, 
We have an issue with an approximative search terms. In our autocomplete result list we have exactly what we expected, but in the the search result page, we don't have any result.
Our configuration is made to obtain exactly the same result in the autocomplete list and in the search result page.
It works very well for most of 95% of our test cases but not for approximative results.

For example, we tape in french "adhesif transpa" instead of "adhesif transparent".

### Preconditions

Magento Version : 2.3.2

ElasticSuite Version : 2.8.0

Environment :Production

Third party modules :N/A

### Cconfirguration

**Relevance Configuration:**

1.  Fulltext Base Settings 
- Minimum Should Match : 100%
- Tie Breaker : 1
2.  Phrase Match Configuration
- Enable Boost on Phrase Match : Yes
- Phrase Match Boost Value : 10
3. Cutoff Frequency Configuration
- Cutoff Frequency : 0.15

**Spellchecking Configuration**
1. Search Fuzziness Configuration
- Enable : Yes
- Value : "Automatic"
- Prefix length : 0
- Max Expansion : 10
2. Phonetic Search Configuration
- Enable : Yes

 **Thesaurus Configuration**
1. General 
- Max allowed Rewrites : 2
2. Synonyms Configuration
- Enable Synonyyms Search : Yes
- Synonyms weight divider : 10
3. Query Expansion Configuration
- Enable Search Expansions : Yes
- Concept weight divider : 10

For your information, Optimizers List and Thesaurus are not created yet.

### Steps to reproduce

1. Search with autocomplete smile
2. Search for an product name with a 2nd word unfinished as in french "adhesif transpa" instead of "adhesif transparent"

### Expected result

1. Search page should load with products related to the search
2. Search results should be the same than in the autocomplete results
3. Search page should return results

### Actual result

1. Our autocomplete return all approximate results and also popular search, categories etc... as expected

![search_result](https://user-images.githubusercontent.com/12586404/63990048-3e8ef080-cb2e-11e9-831e-e238e01fa7f1.png)

2. Our search result page return nothing

![recherche_rapide](https://user-images.githubusercontent.com/12586404/63990051-4189e100-cb2e-11e9-98b4-a0e947b90dea.png)

**Comment by romainruaud** [comment]
TLDR : it's by design. Duplicate of #970 #1036 #1277 

Detailed explanations are on this note : https://github.com/Smile-SA/elasticsuite/issues/1277#issuecomment-456385530
